xml/System.Net.Http/HttpVersionPolicy.xml

```diff
@@ -54,7 +62,17 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Use the highest available version, downgrading only to the requested version but not below. Throws an <see cref="HttpRequestException" /> if a connection with higher or equal version cannot be established.</summary>
```

Comment:
Same here with exception

Reply:
Same, it's just an enum value, so nothing to do here.